configs/default/GEPR-GEPRCF745_BT_HD.config

```diff
@@ -0,0 +1,153 @@
+# Betaflight / STM32F745 (S745) 4.1.0 Sep 25 2019 / 01:23:16 (064ca9b75) MSP API: 1.42
+
+board_name GEPRCF745_BT_HD
+manufacturer_id GEPR
+
+# resources
+resource BEEPER 1 D02
+resource MOTOR 1 B00
+resource MOTOR 2 B01
+resource MOTOR 3 B05
+resource MOTOR 4 B04
+resource MOTOR 5 D12
+resource MOTOR 6 D13
+resource MOTOR 7 C08
+resource MOTOR 8 C09
+resource LED_STRIP 1 A08
+resource SERIAL_TX 1 A09
+resource SERIAL_TX 2 A02
+resource SERIAL_TX 3 B10
+resource SERIAL_TX 4 A00
+resource SERIAL_TX 6 C06
+resource SERIAL_TX 7 E08
+resource SERIAL_TX 8 E01
+resource SERIAL_RX 1 A10
+resource SERIAL_RX 2 A03
+resource SERIAL_RX 3 B11
+resource SERIAL_RX 4 A01
+resource SERIAL_RX 6 C07
+resource SERIAL_RX 7 E07
+resource SERIAL_RX 8 E00
+resource I2C_SCL 1 B08
+resource I2C_SCL 2 B10
+resource I2C_SDA 1 B09
+resource I2C_SDA 2 B11
+resource LED 1 C13
+resource SPI_SCK 1 A05
+resource SPI_SCK 2 B13
+resource SPI_SCK 3 C10
+resource SPI_SCK 4 E02
+resource SPI_MISO 1 A06
+resource SPI_MISO 2 B14
+resource SPI_MISO 3 C11
+resource SPI_MISO 4 E05
+resource SPI_MOSI 1 A07
+resource SPI_MOSI 2 B15
+resource SPI_MOSI 3 C12
+resource SPI_MOSI 4 E06
+resource CAMERA_CONTROL 1 B03
+resource ADC_BATT 1 C03
+resource ADC_RSSI 1 C05
+resource ADC_CURR 1 C02
+resource ADC_EXT 1 C01
+resource SDCARD_CS 1 A15
+resource PINIO 1 E13
+resource PINIO 2 C14
+resource OSD_CS 1 E04
+resource GYRO_EXTI 1 D00
+resource GYRO_EXTI 2 D08
+resource GYRO_CS 1 A04
+resource GYRO_CS 2 B12
+
+# timer
+timer A08 AF1
+# pin A08: TIM1 CH1 (AF1)
+timer B03 AF1
+# pin B03: TIM2 CH2 (AF1)
+timer B00 AF2
+# pin B00: TIM3 CH3 (AF2)
+timer B01 AF2
+# pin B01: TIM3 CH4 (AF2)
+timer B04 AF2
+# pin B04: TIM3 CH1 (AF2)
+timer B05 AF2
+# pin B05: TIM3 CH2 (AF2)
+timer D12 AF2
+# pin D12: TIM4 CH1 (AF2)
+timer D13 AF2
+# pin D13: TIM4 CH2 (AF2)
+timer C08 AF3
+# pin C08: TIM8 CH3 (AF3)
+timer C09 AF3
+# pin C09: TIM8 CH4 (AF3)
+
+# dma
+dma ADC 1 0
+# ADC 1: DMA2 Stream 0 Channel 0
+dma pin A08 0
+# pin A08: DMA2 Stream 6 Channel 0
+dma pin B03 0
+# pin B03: DMA1 Stream 6 Channel 3
+dma pin B00 0
+# pin B00: DMA1 Stream 7 Channel 5
+dma pin B01 0
+# pin B01: DMA1 Stream 2 Channel 5
+dma pin B04 0
+# pin B04: DMA1 Stream 4 Channel 5
+dma pin B05 0
+# pin B05: DMA1 Stream 5 Channel 5
+dma pin D12 0
+# pin D12: DMA1 Stream 0 Channel 2
+dma pin D13 0
+# pin D13: DMA1 Stream 3 Channel 2
+dma pin C08 1
+# pin C08: DMA2 Stream 4 Channel 7
+dma pin C09 0
+# pin C09: DMA2 Stream 7 Channel 7
+
+# feature
+feature -RX_PARALLEL_PWM
+feature -AIRMODE
+feature RX_SERIAL
+feature TELEMETRY
+feature LED_STRIP
+feature OSD
+feature ESC_SENSOR
+
+# serial
+serial 0 1 115200 57600 0 115200
+serial 1 64 115200 57600 0 115200
+serial 2 1 115200 57600 0 115200
+serial 7 1024 115200 57600 0 115200
+
+# aux
+aux 0 0 0 900 1300 0 0
+aux 1 1 1 1700 2100 0 0
+aux 2 28 1 900 1300 0 0
+aux 3 40 4 900 1300 0 0
```

Comment:
what are these for?

Reply:
Hello @DusKing1 . These aux settings are mainly used to configure the use of Bluetooth and digital switches for user priority. So it should be kept here.